### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by Lyfts**
I felt compelled to do this after noticing this casting hell https://github.com/GTNewHorizons/WitchingGadgets/blob/152d118be35a7074bbc5fea24cf83c649bd483a6/src/main/java/witchinggadgets/common/items/ItemInfusedGem.java#L674-L721

It's mostly surface level stuff so nothing should break. (i hope)

### CODE REVIEW

File: src/main/java/witchinggadgets/client/ClientEventHandler.java [hunk 1]
```diff
@@ -229,24 +227,17 @@ public void renderGameOverlay(RenderGameOverlayEvent.Pre event) {
                         GL11.glDisable(GL11.GL_LIGHTING);
                         GL11.glDepthMask(false);
                         ClientUtilities.bindTexture("witchinggadgets:textures/models/white.png");
-                        GL11.glEnable(3042);
                         GL11.glEnable(GL11.GL_BLEND);
-                        GL11.glBlendFunc(770, 771);
+                        GL11.glEnable(GL11.GL_BLEND);
```
Comment (Ethryan) [inline src/main/java/witchinggadgets/client/ClientEventHandler.java:231]: Double enable of GL Blend?

File: src/main/java/witchinggadgets/client/render/TileRenderWallMirror.java [hunk 2]
```diff
@@ -53,8 +53,8 @@ public void renderTileEntityAt(TileEntityWallMirror tile, double x, double y, do
 
         Tessellator tes = Tessellator.instance;
 
-        // GL11.glEnable(3042);
-        // GL11.glBlendFunc(770, 771);
+        // GL11.glEnable(GL11.GL_BLEND);
```
Comment (Ethryan) [inline src/main/java/witchinggadgets/client/render/TileRenderWallMirror.java:56]: Delete this?

File: src/main/java/witchinggadgets/client/ClientEventHandler.java (hunk 1 above)
Comment (Lyfts) [reply src/main/java/witchinggadgets/client/ClientEventHandler.java:231]: Yeah idk it was like that before but applyDecompilerCleanupToMain has revealed it.

File: src/main/java/witchinggadgets/client/render/TileRenderWallMirror.java (hunk 2 above)
Comment (Lyfts) [reply src/main/java/witchinggadgets/client/render/TileRenderWallMirror.java:56]: I also didn't touch that, applyDecompilerCleanupToMain did but sure I can go back and delete it

File: src/main/java/witchinggadgets/common/blocks/tiles/TileEntityThaumicApplicator.java
Comment (Ethryan) [inline src/main/java/witchinggadgets/common/blocks/tiles/TileEntityThaumicApplicator.java:1]: Delete the file?

Comment (Lyfts) [reply src/main/java/witchinggadgets/common/blocks/tiles/TileEntityThaumicApplicator.java:1]: I did delete it?

**Review by Ethryan — COMMENTED** [review]
Otherwise I can’t see anything in need of a change.

File: src/main/java/witchinggadgets/common/blocks/tiles/TileEntityThaumicApplicator.java
Comment (Ethryan) [reply src/main/java/witchinggadgets/common/blocks/tiles/TileEntityThaumicApplicator.java:1]: My bad, I must have touched extend by accident.

**Review by Ethryan — APPROVED** [review]
Looks good

**Pull request merged**